bokeh/core/json_encoder.py

```diff
@@ -49,7 +49,7 @@
 pd = import_optional('pandas')
 rd = import_optional("dateutil.relativedelta")
 
-NP_EPOCH = np.datetime64('1970-01-01T00:00:00Z')
+NP_EPOCH = np.datetime64(0, 's')
```

Comment:
Bokeh serializes to all date time values to FP ms since epoch. Would that just be `np.datetime64(0, 'ms')`? Agreed arithmetic probably makes things work but no reason not to be precise.